bokeh/crossfilter/models.py

```diff
@@ -238,83 +439,208 @@ def make_xy_facet_plot(self):
                     tools="pan,wheel_zoom"
                 )
                 plot.min_border = 0
-                plot.border_symmetry = "none"
+                plot.v_symmetry = False
+                plot.h_symmetry = False
                 row.append(plot)
+
+            # append the row to the list of rows
             grid_plots.append(row)
-        grid = GridPlot(children=grid_plots, plot_width=200 * len(all_facets_x))
-        return grid
 
+        # return the grid of plots as the plot
+        grid = GridPlot(children=grid_plots, plot_width=200*len(all_facets_x))
+        return grid
 
     def make_single_plot(self, df=None, title=None,
                          plot_width=700, plot_height=680,
-                         tools="pan,wheel_zoom,box_zoom,save,resize,select,reset"
-                     ):
+                         tools="pan,wheel_zoom,box_zoom,save,resize,"
+                               "box_select,reset"):
```

Comment:
You can leave all in one line here. For readability, I think it is better so have one line, even if we are above the 80 chars...